app/views/pages/reviews/create.liquid

```diff
@@ -0,0 +1,26 @@
+---
+method: post
+---
+{% liquid
+  include 'lib/must_be_authenticated'
+
+  include 'lib/commands/reviews/create/build', object: context.params.item
+  include 'lib/commands/reviews/create/check', object: context.exports.object
+
+  if context.exports.object.valid
```

Comment:
as a syntax sugar we have added `function_rc` and `return_rc` which will make it easier to maintain the code - you can take a look at app/views/pages/sessions/create.liquid for example - this concept replaces `context.exports`